test/middleware-mockserver.js

```diff
@@ -101,7 +101,7 @@ var mocks = {
 		} else if ( req.query.callback ) {
 			callback = Promise.resolve( req.query.callback );
 		} else if ( req.method === "GET" ) {
-			callback = Promise.resolve( req.url.match( /^.+\/([^\/?.]+)\?.+$/ )[ 1 ] );
+			callback = Promise.resolve( req.url.match( /^.+\/([^\/?]+)\?.+$/ )[ 1 ] );
```

Comment:
Note to self: check why one test is using the file name on the server as the callback name as these should be unrelated. Maybe there's a better way to fix this.